lost/src/main/java/com/mapzen/android/lost/internal/GeofencingApiImpl.java

```diff
@@ -70,12 +72,16 @@ public void disconnect() {
   private PendingResult<Status> addGeofence(LostApiClient client, Geofence geofence,
       PendingIntent pendingIntent) throws SecurityException {
 
-    internalIntent = intentFactory.createIntent();
-    internalIntent.putExtra(GeofencingIntentService.EXTRA_PENDING_INTENT, pendingIntent);
+    int pendingIntentId = (int) System.currentTimeMillis();
```

Comment:
We could but it might not be worth it because this id needs to be an integer. It's used when we call `PendingIntent getService(Context context, int requestCode, Intent intent, int flags)`. The `geofence.getRequestId()` is a string so we'd have to check for an exception when converting the string to an integer.